ga4gh/frontend.py

```diff
@@ -66,9 +75,41 @@ def handle_error(exception):
     return response
 
 
+def getUrls():
+    urls = []
+    rules = app.url_map.iter_rules()
+    excluded_methods = ('OPTIONS', 'HEAD')
+    excluded_rules = (
+        '/', '/flask-api/static/<path:filename>', '/static/<path:filename>')
+    for rule in rules:
+        for method in rule.methods:
+            if method not in excluded_methods and \
```

Comment:
Use parentheses in favour of line breaks here: https://www.python.org/dev/peps/pep-0008/#maximum-line-length